script/gen_client.sh

```diff
@@ -17,7 +17,7 @@
 
 location=$(dirname $0)
 
-unset GOPATH
+#unset GOPATH
```

Comment:
If it not required, we better remove even the comment IMO

Reply:
done